manager/src/main/java/com/redhat/service/smartevents/manager/dao/BridgeDAO.java

```diff
@@ -56,4 +60,12 @@ public ListResult<Bridge> findByCustomerId(String customerId, QueryResourceInfo
 
         return new ListResult<>(bridges, queryInfo.getPageNumber(), total);
     }
+
+    public Long countActiveBridge(String orgId, QuotaType instanceType) {
+        TypedQuery<Long> namedQuery = getEntityManager().createNamedQuery("BRIDGE.countActiveBridgeByOrgAndInstanceType", Long.class);
+        namedQuery.setParameter("organisationId", orgId);
```

Comment:
@vaibhavjainwiz Could we make these constants in the same way as `Bridge.CUSTOMER_ID_PARAM`. 

Thanks!

Reply:
done